msticpy/datamodel/entities/alert.py

```diff
@@ -71,25 +83,81 @@ def __init__(self, src_entity: Mapping[str, Any] = None, **kwargs):
         self.DisplayName: Optional[str] = None
         self.CompromisedEntity: Optional[str] = None
         self.Count: Any = None
-        self.StartTimeUtc: datetime = datetime.min
```

Comment:
Are we breaking V3 compatability here.
Seems that Start and EndttimeUtc are the official properties. https://dev.azure.com/msazure/One/_git/Rome-Detection-AlertContracts?path=/doc/AlertV3.md&_a=preview

Perhaps we should have both - i.e. allow the same value to be set from either name and return either the Utc or non-Utc suffixed versions.